spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -288,6 +292,11 @@ sealed class SpdxSingleLicenseExpression : SpdxExpression() {
      * Return the license exception identifier if this is a [SpdxLicenseWithExceptionExpression] or null otherwise.
      */
     abstract fun exception(): String?
+
+    /**
+     * Return the URL to spdx.org for this license.
+     */
+    abstract fun getLicenseUrl(): String
```

Comment:
How about making the return value nullable? I think it's a bit cleaner if the caller can check if the URL is null than having to check if the return string is blank.

Reply:
Given that the SPDX license list only contains direct entries for licenses *without* exceptions, shouldn't this function be moved to `SpdxLicenseIdExpression`?

Reply:
True, it's not required in `SpdxExpression`, but I think `SpdxSingleLicenseExpression` would be a more convenient place.

Reply:
Now it's nullable, and kept at `SpdxSingleLicenseExpression`.